Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Taxonomy of Failsafe Levels
+type: Decision Record
+status: Draft
+track: IaaS
+---
+
+
+## Abstract
+
+When talking about redundancy and backups in the context of cloud infrastructures, the scope under which circumstances these concepts apply to various ressources is neither homogenous nor intuitive.
+This decision record aims to define different levels of failure-safety.
+These levels can then be used in standards to clearly set the scope that certain procedures in e.g. OpenStack offer.
+
+## Glossary
+
+| Term               | Explanation                                                                                                                              |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| Virtual Machine    | Equals the `server` resource in Nova.                                                                                                    |
+| Ironic Machine     | A physical node managed by Ironic or as a `server` resource in Nova.                                                                     |
+| Ephemeral Storage  | Disk storage directly supplied to a virtual machine by Nova. Different from volumes.                                                     |
+| (Glance) Image     | IaaS resource usually storing raw disk data. Managed by the Glance service.                                                              |
+| (Cinder) Volume    | IaaS resource representing block storage disk that can be attached as a virtual disk to virtual machines. Managed by the Cinder service. |
+| (Volume) Snapshot  | Thinly-provisioned copy-on-write snapshots of volumes. Stored in the same Cinder storage backend as volumes.                             |
+| Volume Type        | Attribute of volumes determining storage details of a volume such as backend location or whether the volume will be encrypted.           |
+| (Barbican) Secret  | IaaS resource storing cryptographic assets such as encryption keys. Managed by the Barbican service.                                     |
+| Key Encryption Key | IaaS resource, used to encrypt other keys to be able to store them encrypted in a database.                                              |
+| Floating IP        | IaaS resource, an IP that is usually routed and accessible from external networks.                                                       |
+| Disk               | A physical disk drive (e.g. HDD, SSD) in the infrastructure.                                                                             |
+| Node               | A physical machine in the infrastructure.                                                                                                |
```

Comment:
If we list hardware resources, does it make sense to mention network node as an additional main single-point-of-failure?

```suggestion
| Node               | A physical machine in the infrastructure providing computational and/or storage capabilities.                                                                                                | Network node | A physical device providing network connectivity, such as switch or router. 
```